pkg/grafana/folders.go

```diff
@@ -13,6 +13,16 @@ import (
 
 // getRemoteFolder retrieves a folder object from Grafana
 func getRemoteFolder(uid string) (*grizzly.Resource, error) {
+	h := FolderHandler{}
+	if uid == "General" || uid == "general" {
+		folder := Folder{
+			"id":    0.0,
```

Comment:
Is there a reason for the id to be float?

Reply:
simply because that's what the JSON lib expects.